### ISSUE TITLE
⌨ "Toggle Column Visible" title read instead of checkbox state for admin tables

### ISSUE THREAD

**Issue by gobyrne** [GCTC-NTGC/gc-digital-talent#7274]
## 🕵️‍♀️ Details

**AT Configuration**
NVDA with Chrome on Windows

Tester Feedback

-"When in the columns dialogue to hide them, checkboxes are overly verbose if I jump from checkbox to checkbox with NVDA. every time I jump to the next or previous checkbox, it will read the label of the checkbox but add "Toggle Column Visible" which makes it overly verbose."

**Related WCAG criteria**
[2.4.6 Headings and Labels](https://w3c.github.io/wcag/understanding/headings-and-labels) 

## Screenshot

![Image](https://user-images.githubusercontent.com/101275838/253133776-a3bd12fe-22fe-46f9-b605-421d94753667.png)

**💻 Related WCAG Criteria**

## 🧪 Testing Steps

- Access the pools table from the admin UI using NVDA
- Navigate using the keyboard to the Columns button.
- When in the modal navigate to an unchecked (or hidden) box. 
- Observe what NVDA read.

## 🌟 Acceptance Criteria
- [ ] Remove the Title attribute from the checkboxes. The given title is currently getting read INSTEAD of the name and state of the actual checkbox.
- [ ]  This should be done for all similar tables.
